config/eslintrc.json

```diff
@@ -1,8 +1,11 @@
 {
-    "parser": "babel-eslint",
+    "parser": "@typescript-eslint/parser",
     "extends": [
         "eslint:recommended",
-        "airbnb"
+        "airbnb",
+        "plugin:@typescript-eslint/recommended",
+        "prettier",
+        "prettier/@typescript-eslint"
```

Comment:
I have seen in https://github.com/prettier/eslint-config-prettier that there also is a `prettier/react`. No idea what that changes and whether that also might be helpful.

Reply:
I can't see what that changes either really; Prettier is working fine with React as it is so I don't think we need to add anything else.